### ISSUE TITLE
[bug]: should modified_at in TableMetadata be null until it is modified?

### ISSUE THREAD

**Issue by signekb** [seedcase-project/seedcase-sprout#122]
### Steps to reproduce

In `models.py`, the field `modified_at` is set to `auto_now=True`. I would argue that this field should be Null until it is actually modified. 

### Expected result

I would expect `modified_at` to be Null when it hasn't been modified, but only created.

### Actual result (the bug)

Currently, when you create a new row in `TableMetadata`, `modified_at` is equal to `created_at`. 

### Severity/Priority

3: Minor

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by K-Beicher** [comment]
I don't really have a strong opinion on this.  I agree that it should be NULL, but when I've used fields like that in the past I've just ignored all the ones where the two timestamps were the same, so... 

tl;dr
It should be fixed, but it is quite a low priority for me.

**Comment by philter87** [comment]
Agreed that it should be NULL, but... It's very low priority. I prefer cleaner or less code. 

I will take a look and see if there is an easy fix

**Comment by philter87** [comment]
My point was, that we need to remember it when we change something in TableMetadata like the table name. 
But... this was actually not necessary as I overrode the save method and then it happens automatically
